docs/installation.rst

```diff
@@ -78,7 +56,24 @@ this method by following the steps described below.
       .. note::
 
         The complete developer guidelines can be found :ref:`here <developer_guidelines>`.
+        
+    b. Non-developers can install from specific releases using pip-
+
+      ::
 
+        $ pip install git+https://github.com/tardis-sn/tardis.git@{tag}
+
+      For example- 
+
+      ::
+      
+        $ pip install git+https://github.com/tardis-sn/tardis.git@release-2023.09.17
```

Comment:
The release tag chosen needs to explicitly match the release tag of the environment file to guarantee compatibility. This should be made very clear in the docs.

Reply:
```suggestion
        $ pip install git+https://github.com/tardis-sn/tardis.git@release-<yyyy.mm.dd> 
        
        Please fill in the year, month, and date of the released version you want to use.
```